diff --git a/app/src/usb_hid.c b/app/src/usb_hid.c
index 9adc7db8..1e0c5bea 100644
--- a/app/src/usb_hid.c
+++ b/app/src/usb_hid.c
@@ -40,7 +40,7 @@ static void in_ready_cb(const struct device *dev) { k_sem_give(&hid_sem); }
 #define HID_REPORT_TYPE_FEATURE 0x300
 
 #if IS_ENABLED(CONFIG_ZMK_USB_BOOT)
-static uint8_t hid_protocol = HID_PROTOCOL_REPORT;
+static uint8_t hid_protocol = HID_PROTOCOL_BOOT;
 
 static void set_proto_cb(const struct device *dev, uint8_t protocol) { hid_protocol = protocol; }
 
